src/BrazeKit-dev.js

```diff
@@ -512,6 +592,24 @@ var constructor = function() {
     this.onUserIdentified = onUserIdentified;
     this.removeUserAttribute = removeUserAttribute;
     this.decodeClusterSetting = decodeClusterSetting;
+
+    function kitLogger(method) {
```

Comment:
Can you provide a comment of the expected output of this function so future devs can use it as a cheat sheet?

Reply:
good call, will do